week3/HackYourWeather1/hackyourhomework/src/components/Cities.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import CityBox from './CityBox';
+
+const Cities = props => {
+  return (
+    <div>
+      {props.data.map((city, index) => {
+        return <CityBox data={city} key={index} id={index} click={props.handle} />;
```

Comment:
city.id is safer than the `index`, if you have it use it.
```suggestion
        return <CityBox data={city} key={index} id={city.id} click={props.handle} />;
```